### DISCUSSION TITLE
Basketball player number of games ahead?

### DISCUSSION THREAD

**Discussion by frank-gentile** [cwendt94/espn-api#394] [ESPN API] [answered]
Is there a way to see the number of games a player on the roster has upcoming? So for example to see that Zach Lavine has 4 games this week or 3 games next week?

Thanks!
Frank

**Comment by cwendt94** [comment, marked as answer]
That feature was just added! Players now have a `schedule` field that is a dict of all the games. You will need to do some filtering because its not ordered but the keys are scoring period and the value is an object with date and team they are playing!
Example
```python3
# Karl-Anthony Towns
league.teams[0].roster[0].schedule
{'131': {'team': 'CHA', 'date': datetime.datetime(2023, 2, 25, 1, 0)}, 
'3': {'team': 'OKC', 'date': datetime.datetime(2022, 10, 20, 0, 0)}, 
'133': {'team': 'GSW', 'date': datetime.datetime(2023, 2, 27, 0, 30)}, 
'5': {'team': 'UTA', 'date': datetime.datetime(2022, 10, 22, 0, 0)}}
```

**Reply by frank-gentile** [reply]
Great- thank you for your timely response!
